modules/setTime.js

```diff
@@ -1,11 +1,9 @@
-/* eslint-disable linebreak-style */
-/* eslint-disable no-trailing-spaces */
 /* eslint-disable no-undef */
```

Comment:
- [ ] Kindly remove all the `eslint-disable` lines.

Reply:
@ShadyShawkat 
Hello sir Hope you are well.
sir please check my repo if you accept then I will submit, because I don't want to give extra review.
and I don't know if I remove one comment so it will give error and I don't know how can I solve this.
so please help me sir.
thanks
the link bellow is my repo so you can check it. thanks again.
https://github.com/ZuhRa-HashiMi/Awesome_books_with_ES6/pull/1